pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/RealtimeSegmentStatsHistory.java

```diff
@@ -323,4 +321,14 @@ private int getNumntriesToScan() {
     }
     return getCursor();
   }
+
+  public static void main(String[] args) throws Exception {
```

Comment:
can be removed?

Reply:
I think it is useful for debugging. We can copy a history file from a server and look at it.